lib/lightning/workflows/presence.ex

```diff
@@ -41,22 +43,23 @@ defmodule Lightning.Workflows.Presence do
   end
 
   @doc """
-  Tracks the presence of a user on a given topic.
+  Tracks the presence of a user on editing a workflow.
 
   ## Parameters
 
     - `user`: The user to be tracked.
-    - `topic`: The topic to track the user on.
+    - `workflow`: The workflow to track the user on.
     - `pid`: The process identifier for the user's session.
 
   ## Examples
 
-      iex> Lightning.Workflows.Presence.track_user_presence(%User{id: 1}, "room:lobby", self())
+      iex> Lightning.Workflows.Presence.track_user_presence(%User{id: user_id}, %Workflow{id: workflow_id}, self())
       :ok
 
   """
-  def track_user_presence(user, topic, pid) do
+  def track_user_presence(user, %Workflow{id: workflow_id}, pid) do
```

Comment:
Nicely done 🙌 . I was worried for a second then later noticed that it's already scoped for `workflows`